### DISCUSSION TITLE
Does Spegel work on k3s?

### DISCUSSION THREAD

**Discussion by bojanraic** [onedr0p/cluster-template#1458] [Q&A] [answered]
Basically, the title. In my k3s config file, it is supposed to be enabled, yet it doesn't seem to work, at least not how I understand it should work.

I noticed spegel helm repo was (still is?) part of the template, so I am wondering if it was supposed to be installed explicitly until k3s started bundling it in.
I know you're considering winding down k3s support, but I would still like to know if Spegel works out of the box or at least when installed explicitly via Helm.
Thanks!

**Comment by onedr0p** [comment, marked as answer]
k3s has spegel embedded now and it is enabled by default in this repo. Whether it works or not would be up to people to report back and help provide any fixes, or open issues upstream if they run into issues.

https://docs.k3s.io/installation/registry-mirror
